mp/src/game/client/momentum/ui/HUD/hud_speedometer_label.cpp

```diff
@@ -0,0 +1,285 @@
+#include "cbase.h"
+
+#include "hud_speedometer_label.h"
+
+#include <vgui_controls/AnimationController.h>
+
+#include "hud_comparisons.h"
+#include "iclientmode.h"
+#include "momentum/util/mom_util.h"
+#include "mom_player_shared.h"
+
+#include "tier0/memdbgon.h"
+
+// 1 unit = 19.05mm -> 0.01905m -> 0.00001905Km(/s) -> 0.06858Km(/h)
+#define UPS_TO_KMH_FACTOR 0.06858f
+// 1 unit = 0.75", 1 mile = 63360. 0.75 / 63360 ~~> 0.00001184"(/s) ~~> 0.04262MPH
+#define UPS_TO_MPH_FACTOR 0.04262f
+
+// velocity has to change this much before it is colored as increase/decrease
+#define COLORIZE_DEADZONE 2.0f
+
+using namespace vgui;
+
+SpeedometerLabel::SpeedometerLabel(Panel *parent, const char *panelName, SpeedometerColorize_t colorizeType)
+    : Label(parent, panelName, ""), m_eColorizeType(colorizeType), m_pFlAlpha(nullptr),
+      m_pSzAnimationName(nullptr), m_bSupportsEnergyUnits(false), m_bDoneFading(false),
+      m_eUnitType(SPEEDOMETER_UNITS_UPS), m_pRangeList(nullptr), m_bDrawComparison(true), 
+      m_bSupportsSeparateComparison(true), m_cvarGravity("sv_gravity")
```

Comment:
Instead of using a cvar ref for gravity, do:
```
#include "movevars_shared.h"

. . . 

using namespace vgui;

extern ConVar sv_gravity;
```

And use `sv_gravity` directly.